instrumentation/mybatis/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/mybatis/MybatisSingletons.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.mybatis;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanNameExtractor;
+
+public final class MybatisSingletons {
```

Comment:
would it be better to name this class `MyBatisSingletons` to follow how mybatis is usually written?

Reply:
> would it be better to name this class `MyBatisSingletons` to follow how mybatis is usually written?

I adjusted it.